05-pagination-querying.md

```diff
@@ -0,0 +1,366 @@
+# Pagination, Querying and Sorting
+
+## What is pagination?
+
+Pagination of data is a way to limit the output of a list of resources to a well-defined (maximum) size.
+This paginated list is provided to the consumer of the REST api as a result of a query on a certain resource.
+
+Instead of returning a complete list of records, only a fraction of the data is returned with information necessary to retrieve the rest of the data.
+The result is often wrapped in an envelope that contains the following data:
+* The number of the current page
+* The total number of pages
+* The content of this page, which is a list of elements.
+
+## Why paginate data?
```

Comment:
Introduce a bullet-point list here to summarize the reasons. Then drill down into them as you do now.